lib/portlayer/network/context.go

```diff
@@ -113,6 +114,9 @@ func NewContext(bridgePool net.IPNet, bridgeMask net.IPMask, config *Configurati
 		}
 	}
 
+	// subscribe to the event stream for Vm events
+	exec.Config.EventManager.Subscribe(events.NewEventType(events.ContainerEvent{}).Topic(), "network", ctx.handleEvent)
```

Comment:
This is created on a per-context basis which implies we could have more than one context per VCH?  If so, the second arg to subscribe needs to be unique per context -- otherwise only the latest context would receive the callback.